core/pom.xml

```diff
@@ -482,6 +483,24 @@ THE SOFTWARE.
       <artifactId>junit</artifactId>
       <scope>test</scope>
     </dependency>
```

Comment:
If we want to migrate our test base, we should remove this library all together.

Reply:
Thanks for pointing that out, I removed junit4 and added the vintage engine